lib/apiservers/engine/backends/utils/endpoint.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package utils
```

Comment:
We've managed to avoid having a package name 'utils' so far, though we do have a 'util' package.  If we can't find an existing package or better name, let's at least stick with 'util'.  For example, you could name the package 'endpoint' and function 'Alias', then callers use `endpoint.Alias(...)`